lib/booktypecontrol/views.py

```diff
@@ -403,6 +403,22 @@ def get_success_url(self):
         messages.success(self.request, _('Group successfully deleted.'))
         return "%s#list-of-groups" % reverse('control_center:settings')
 
+    def delete(self, request, *args, **kwargs):
+        group = self.get_object()
+
+        # remove books from group
+        for book in group.book_set.all():
+            book.group = None
+            book.save()
```

Comment:
@olegpshenichniy This looks good but, I would only change the loop with multiple save() calls with something like ```group.book_set.update(group=None)``` and I think it will work and use just one query :) Do the change and merge it yourself if you want